src/common/components/help/help-install-snap.js

```diff
@@ -12,6 +12,12 @@ export default class HelpInstallSnap extends Component {
     const revOption = revision ? `--revision=${ revision }` : '';
     const command = children || `sudo snap install --edge ${name} ${revOption}`;
 
+    /**
+    const tweet = `Install ${name} in seconds on Linux OSes:\n`
```

Comment:
It would be nice to have a reference to #655 in the comment to remember why it's commented out.